test/test_dataloader.py

```diff
@@ -294,6 +294,26 @@ def test_default_colate_bad_numpy_types(self):
         arr = np.array([[[object(), object(), object()]]])
         self.assertRaises(TypeError, lambda: default_collate(arr))
 
+    def test_sparse_tensor_loading(self):
```

Comment:
Should I write a different test for this? I think being able to serialize sparse tensors in multiprocessing isn't a dataloader-specific thing

Reply:
I'm fine with either way. You could add a test that puts a sparse tensor in a `multiprocessing.Queue` and takes it out.

Reply:
Thanks!